src/kili/adapters/kili_api_gateway/helpers/queries.py

```diff
@@ -102,36 +87,36 @@ def execute_query_from_paginated_call(
                     if len(elements) < first:
                         break
 
+    def get_number_of_elements_to_query(
+        self,
+        count_query: Optional[str],
```

Comment:
I find it weird to make it optional

if we call this method, we must provide the query (if it exists), else it makes no sense to call it

Reply:
I agree, changed

Reply:
it's still optional?